openpype/tools/publisher/window.py

```diff
@@ -400,6 +403,7 @@ def closeEvent(self, event):
         # TODO capture changes and ask user if wants to save changes on close
         if not self._controller.host_context_has_changed:
             self._save_changes(False)
+        self._comment_input.setText("")  # clear comment
```

Comment:
Should it be cleared on close?

Reply:
It got this reported from an artist, and I'd say yes. If it didn't it would also mean that when you reopen it later, which could be by the time you're working in another workfile in another shot then it'd still have the much older comment.

Aside of what the artist reported, I'd say so too, yes. It's weird that closing the UI and reopening it doesn't start it fresh I'd say.